x-pack/plugins/ml/public/jobs/new_job/simple/multi_metric/create_job/create_job_controller.js

```diff
@@ -465,7 +512,12 @@ module
           })
           .catch((resp) => {
             // save failed
-            msgs.error('Save failed: ', resp.resp);
+            msgs.error(
+              i18n('xpack.ml.newJob.simple.multiMetric.saveFailedErrorMessage', {
+                defaultMessage: 'Save failed: {response}',
+                values: { response: resp.resp }
```

Comment:
Not sure that we need to move resp.resp to values. It looks like it should be still put as second param to msgs.error